src/components/ec-timer/ec-timer.vue

```diff
@@ -74,22 +89,40 @@ const props = defineProps({
 });
 const emit = defineEmits(['time-expired']);
 
-const diameter = computed(() => radius.value * 2 + strokeWidth.value);
+const radius = 24;
+const strokeWidth = 4;
+const diameter = computed(() => radius * 2 + strokeWidth);
 const viewbox = computed(() => `0 0 ${diameter.value} ${diameter.value}`);
-const circumference = computed(() => 2 * Math.PI * radius.value);
+const circumference = computed(() => 2 * Math.PI * radius);
```

Comment:
This doesn't have to be computed prop if there's nothing reactive used in it.

Reply:
if i remove the reactivity the blu circle stops to reduce itself

Reply:
It's a multiplication of 3 constants so I can't have any effect. Did you clean up all `.value`s from anywhere? I tried this on my local and it worked for me (there's lots of things that can be removed):
```js
const radius = 24;
const strokeWidth = 4;
const diameter = radius * 2 + strokeWidth;
const viewbox = `0 0 ${diameter} ${diameter}`;
const circumference = 2 * Math.PI * radius;
const steps = computed(() => circumference / props.seconds);

const offset = computed(() => {
  if (props.isRunning) {
    return circumference + steps.value * totalSecondsLeft.value;
  }
  return circumference;
});

```

Reply:
Had you a TS error about that claimed about missing `value` property in `number` type?

Reply:
you mean in ebo?

Reply:
no, no errors

Reply:
No, I forgot that this component is not in TS yet :). Otherwise if you hadn't remove `.value` you'd have an error from TS that `circumReference` has no `value` prop, as it is type of `number` after removing `computed`.

Reply:
yes i meant i had not errors because the file is still in js